src/App.css

```diff
@@ -1,38 +1,92 @@
 .App {
-  text-align: center;
+  display: grid;
+  grid-template-rows: min-content 1fr;
+  gap: var(--spacing--gap);
+  padding: var(--spacing--gap);
+  width: 100vw;
+  height: 100vh;
+}
+
+.App-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: var(--spacing--gap);
 }
 
 .App-logo {
-  height: 3em;
-  padding: .5em;
+  aspect-ratio: 4.2;
 }
 
-.React-logo {
-  height: 50vh;
-  width: 50vw;
+.App-header .App-logo {
+  height: 60px;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .React-logo {
-    animation: React-logo-spin infinite 20s linear;
-  }
+.App-navbar {
+  font-size: 0.875rem;
+  color: black;
+  text-transform: uppercase;
+  font-weight: 700;
 }
 
-.App-header {
+.App-navbar ul {
+  display: flex;
   align-items: center;
-  background-color: #efefef;
+  gap: calc(var(--spacing--gap) * 2);
+}
+
+.App-navbar__item {
+  all: unset;
+
   display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  color: black;
+  align-items: top;
```

Comment:
My browser dev tools does not like `top` as a value for [`align-items`](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items); did you mean `start` or `flex-start`?

Reply:
@jasonaowen Ah, I did mean `flex-start`! I thought I'd fixed that, but maybe I habit-typed it again. Good catch!